include/isc_nav/utility/point.hpp

```diff
@@ -44,14 +44,14 @@ inline bool operator== ( const Point3D& lhs, const Point3D& rhs )
     return ( lhs.x == rhs.x && lhs.y == rhs.y && lhs.z == rhs.z );
 }
 
-namespace std
-{
-template<> struct hash<Point2D>
-{
-    [[nodiscard]] constexpr std::size_t operator()(const Point2D& point) const noexcept
-    {
-        return std::hash<size_t>()(static_cast<size_t>(point.x) ^ (static_cast<size_t>(point.y) << 16));
-    }
-};
-} // namespace std
+//namespace std
+//{
+//template<> struct hash<Point2D>
+//{
+ //   [[nodiscard]] constexpr std::size_t operator()(const Point2D& point) const noexcept
```

Comment:
i think if you delete constexpr here it will build and then you wont need to comment it out.

Reply:
that fixed it!